README.md

```diff
@@ -4,6 +4,22 @@ Pore is a plugin for the up-and-coming SpongeAPI built to provide compatibility
 Please note that this project is still under heavy development and **is not yet considered functional**. Currently, no
 Bukkit plugins are verified to be supported.
 
+## Running Pore on Your Sponge Server
+
+If you like living on the bleeding edge and want to give Pore a try:
+
+#. Back up your existing system.
+#. Install [Sponge](https://www.spongepowered.org/), detailed instructions can be found [here](https://docs.spongepowered.org/en/server/getting-started/) .
+#. Copy the [Pore Snapshot](https://ci.caseif.net/job/Pore/lastSuccessfulBuild/artifact/build/libs/Pore-1.0.0-SNAPSHOT.jar) into your server's `mods` directory.
```

Comment:
I'd recommend linking to https://ci.caseif.net/job/Pore/lastSuccessfulBuild/ instead of a direct link.